Sources/PointFreeTestSupport/PointFreeTestSupport.swift

```diff
@@ -24,7 +24,7 @@ extension Environment {
   public static let teamYearly = mock
     |> \.database.fetchSubscriptionTeammatesByOwnerId .~ const(pure([.mock]))
     |> \.database.fetchTeamInvites .~ const(pure([.mock]))
-    |> \.stripe.fetchSubscription .~ const(pure(.teamYearly))
+    |> (\Environment.stripe.fetchSubscription) .~ const(pure(.teamYearly))
```

Comment:
Heh, this snuck in because it sped up compile time here quite a bit.

Reply:
ha I bet